spec.emu

```diff
@@ -279,16 +280,16 @@
         </emu-grammar>
         <emu-alg>
           1. <del>Return ModuleRequests of |FromClause|.</del>
-          1. <ins>Let _specifier_ be SV of the |ModuleSpecifier| contained in |FromClause|.</ins>
-          1. <ins>Return a ModuleRequest Record { [[Specifer]]: _specifier_, [[Attributes]]: « » }.</ins>
+          1. <ins>Let _specifier_ be SV of |ModuleSpecifier|.</ins>
```

Comment:
There doesn't seem to be a `ModuleSpecifier` here, did you mean `FromClause`? Specifically, I think we need `FromClause: from ModuleSpecifier` and `ModuleSpecifier: StringLiteral` cases in this `ModuleRequests` algorithm

Reply:
Yes, I meant `FromClause`.

The problem with `ModuleRequests` for `FromClause` and `ModuleSpeicfier` is that `ModuleRequests` returns a (specifier, attributes) pair, and for `FromClause`/`ModuleSpecifier` you cannot return both. `SV of FromClause` should work here, since SDOs are automatically recursively defined (thus `SV of FromClause` if the `SV of ModuleSpecifier`, which is the `SV of StringLiteral`).